app/Http/Controllers/AdminNgwordController.php

```diff
@@ -11,24 +11,34 @@ class AdminNgwordController extends Controller
     protected $table = 'ng_words';
     private $ngword;
 
-    public function __construct(NGWord $ngword) {
+    public function __construct(NGWord $ngword)
```

Comment:
`private $ngword` is not being used, you can remove this

Also, you can remove the whole `public function __construct`

Reply:
Great work in cleaning the unused properties, @RikoMaezawa31 !